bindings/rust/s2n-tls-sys/templates/Cargo.template

```diff
@@ -45,3 +45,4 @@ cc = { version = "1.0", features = ["parallel"] }
 jobserver = "=0.1.26" # newer versions require rust 1.66, see https://github.com/aws/s2n-tls/issues/4241
 home = "=0.5.5" # newer versions require rust 1.70, see https://github.com/aws/s2n-tls/issues/4395
 regex = "=1.9.6" # newer versions require rust 1.65, see https://github.com/aws/s2n-tls/issues/4242
+zeroize = "=1.7.0" # newer versions require rust 1.72, see https://github.com/aws/s2n-tls/issues/4518
```

Comment:
nit: This isn't really a PR issue, but maybe it would make sense to have a single tracking issue with the pins rather than making a new pin for each one? And just keep the list sorted by "required MSRV"?

Reply:
Yeah that makes sense. I will merge this first though to unblock the CI. https://github.com/aws/s2n-tls/issues/4521